### PR TITLE
Auto updated assets for python

### PR DESCRIPTION
**Pull request by yograjshisode**

### CODE REVIEW

File: python/version.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import argparse
 
 AVI_VERSION = '20.1.5'
```
Comment (chaitanyaavi) [inline python/version.py:7]: This should be 20.1.6

Comment (chaitanyaavi) [inline python/version.py]: This should be 20.1.6

File: python/version.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
-AVI_PIP_VERSION = '20.1.5b1'
+AVI_PIP_VERSION = '20.1.5'
```
Comment (chaitanyaavi) [inline python/version.py]: This should be 20.1.6b1

File: python/avi/migrationtools/ace_converter/test/dummy_input.json [hunk 3]
```diff
@@ -0,0 +1,146 @@
+{
```
Comment (chaitanyaavi) [inline python/avi/migrationtools/ace_converter/test/dummy_input.json]: File name cannot be dummy need to rename file

**Comment by ArpitAVI** [comment]
run jenkins-python-sdk-ut

**Comment by ArpitAVI** [comment]
run jenkins-python-sdk-ut

**Comment by yograjshisode** [comment]
run jenkins-python-sdk-ut

**Comment by yograjshisode** [comment]
run jenkins-python-sdk-ut

**Pull request merged**